tests/e2e/General/UsageTest.php

```diff
@@ -328,15 +328,15 @@ public function testStorageStats(array $data): array
         $this->assertEquals(1, count($response['body']['requests']));
         $this->assertEquals($requestsTotal, $response['body']['requests'][array_key_last($response['body']['requests'])]['value']);
         $this->validateDates($response['body']['requests']);
-        $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
+        $this->assertGreaterThan($storageTotal, $response['body']['filesStorageTotal']);
```

Comment:
This can be confusing reading the test 😬

Reply:
Well what do you suggest?